### PR TITLE
RTT doc additions

### PR DESCRIPTION
**Pull request by carlescufi**

### CODE REVIEW

**Review by IvanSanchez — APPROVED** [review]
LGTM

File: doc/tools/nordic_segger.rst [hunk 1]
```diff
@@ -154,6 +154,14 @@ To use RTT, you will first need to enable it by adding the following lines in yo
    CONFIG_HAS_SEGGER_RTT=y
    CONFIG_RTT_CONSOLE=y
 
+Additionally you might need to disable other consoles which conflict with the
```
Comment (dbkinder) [inline doc/tools/nordic_segger.rst]: Are there symptoms (like missing output) if you don't disable these consoles that users would notice?

File: doc/tools/nordic_segger.rst [hunk 2]
```diff
@@ -154,6 +154,14 @@ To use RTT, you will first need to enable it by adding the following lines in yo
...
+RTT one if they are enabled by default in the particular sample or application
+you are running. For example to disable the UART console:
```
Comment (dbkinder) [inline doc/tools/nordic_segger.rst]: add:

    For example, to disable the UART console, add this to your ``.conf`` file:

**Review by dbkinder — CHANGES_REQUESTED** [review]
one question and a clarification

File: doc/tools/nordic_segger.rst (hunk 1 above)
Comment (carlescufi) [reply doc/tools/nordic_segger.rst]: Yep, I will reword a bit

**Comment by carlescufi** [comment]
@dbkinder addressed

**Review by dbkinder — APPROVED** [review]
thanks

**Pull request merged**